src/main/java/Application.java

```diff
@@ -54,33 +52,35 @@ public static void main(String[] args) {
 
         //Car service
         Car fiatCar = new Car("500", fiatMaker);
-        Car opelCar = new Car("Blitz", fiatMaker);
-        Car tavriaCar = new Car("Slavuta", tavriaMaker);
-        Car ladaCar = new Car("Granta", ladaMaker);
-        final Manufacturer fordMaker = new Manufacturer("Ford", "USA");
+        Car opelCar = new Car("Blitz", opelMaker);
+        final Car tavriaCar = new Car("Slavuta", tavriaMaker);
+        final Car ladaCar = new Car("Granta", ladaMaker);
+        Manufacturer fordMaker = new Manufacturer("Ford", "USA");
+        manufacturerService.create(fordMaker);
         carService.create(fiatCar);
         carService.create(opelCar);
         carService.create(tavriaCar);
         carService.create(ladaCar);
         System.out.println(carService.get(1L).toString());
         System.out.println(carService.get(2L).toString());
         System.out.println(carService.get(3L).toString());
-        fiatCar.setManufacturer(fordMaker);
         carService.update(fiatCar);
```

Comment:
have we changed the fiatCar? is there a reason to update it?

Reply:
No we did not, lost a track. Fixed